src/__tests__/imagings/requests/NewImagingRequest.test.tsx

```diff
@@ -0,0 +1,259 @@
+import { Button, Typeahead, Label, Alert } from '@hospitalrun/components'
+import { mount, ReactWrapper } from 'enzyme'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { act } from 'react-dom/test-utils'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+import createMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import NewImagingRequest from '../../../imagings/requests/NewImagingRequest'
+import * as titleUtil from '../../../page-header/title/useTitle'
+import SelectWithLabelFormGroup from '../../../shared/components/input/SelectWithLableFormGroup'
+import TextFieldWithLabelFormGroup from '../../../shared/components/input/TextFieldWithLabelFormGroup'
+import TextInputWithLabelFormGroup from '../../../shared/components/input/TextInputWithLabelFormGroup'
+import Imaging from '../../../shared/model/Imaging'
+import Patient from '../../../shared/model/Patient'
+import { RootState } from '../../../shared/store'
+
+const mockStore = createMockStore<RootState, any>([thunk])
+
+describe('New Imaging Request', () => {
+  describe('title and breadcrumbs', () => {
+    let titleSpy: any
+    const history = createMemoryHistory()
+
+    beforeEach(() => {
+      const store = mockStore({ title: '', imaging: { status: 'loading', error: {} } } as any)
+      titleSpy = jest.spyOn(titleUtil, 'default')
+      history.push('/imagings/new')
+
+      mount(
+        <Provider store={store}>
+          <Router history={history}>
+            <NewImagingRequest />
+          </Router>
+        </Provider>,
+      )
+    })
+
+    it('should have New Imaging Request as the title', () => {
+      expect(titleSpy).toHaveBeenCalledWith('imagings.requests.new')
+    })
+  })
+
+  describe('form layout', () => {
+    let wrapper: ReactWrapper
+    const history = createMemoryHistory()
+
+    beforeEach(() => {
+      const store = mockStore({ title: '', imaging: { status: 'loading', error: {} } } as any)
+      history.push('/imagings/new')
+
+      wrapper = mount(
+        <Provider store={store}>
+          <Router history={history}>
+            <NewImagingRequest />
+          </Router>
+        </Provider>,
+      )
+    })
```

Comment:
we can use a setup function here as well. 

It would like something like: 


```ts
const setup = () => {
	const store = mockStore({ title: '', imaging: { status: 'loading', error: {} } } as any)
      history.push('/imagings/new')

      wrapper = mount(
        <Provider store={store}>
          <Router history={history}>
            <NewImagingRequest />
          </Router>
        </Provider>,
      )

	return { wrapper: wrapper as ReactWrapper, history, store }
}
```

This will eliminate the need for global variables in this file.